src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c

```diff
@@ -273,8 +252,8 @@ static int fstatat64_wrapper(int dfd, const char *path,
 
 #if defined(__linux__) && defined(_LP64) && defined(__NR_newfstatat)
 #define FSTATAT64_SYSCALL_AVAILABLE
-static int fstatat64_wrapper(int dfd, const char *path,
-                             struct stat64 *statbuf, int flag)
+static int fstatat_wrapper(int dfd, const char *path,
+                             struct stat *statbuf, int flag)
 {
```

Comment:
Minor nit - you can probably fix the align after the edit or collapse it into one line.